Add steps to check and the solution to make KEDA work in proxy network (#83)
kedacore/keda-docs: content/FAQ.md

#### Can KEDA be used in production?
Yes! KEDA is now 1.0 and suited for production workloads.

#### How to make KEDA work in a proxy network?

> If while setting up KEDA, you get an error: "(v1beta1.external.metrics.k8s.io) status FailedDiscoveryCheck" with a Message: no response from https://ip:443: Get https://ip:443: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers); one of the reason for this can be that you are behind a proxy network.

Reviewer: ```suggestion
If while setting up KEDA, you get an error: `(v1beta1.external.metrics.k8s.io) status FailedDiscoveryCheck` with a message: `no response from https://ip:443: Get https://ip:443: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)`.

 One of the reason for this can be that you are behind a proxy network.
```

> 

Reviewer: Nitpick - 2 new lines, let's remove one
```suggestion
```

###### Check the status:

> Find the apiservice name for the service keda/keda-metrics-apiserver:

Reviewer: ```suggestion
Find the api service name for the service `keda/keda-metrics-apiserver`:
```

> ```kubectl get apiservice --all-namespaces```

Reviewer: When using "```" you should use multiple lines, for example:

> \`\`\`
> kubectl get apiservice --all-namespaces
> \`\`\`

Can you change this for all commands please?

Author: Done

> Check for the status of the apiservice found in previous step:

Reviewer: ```suggestion
Check for the status of the api service found in previous step:
```

```kubectl get apiservice <apiservicename' -o yaml```

Example: ```kubectl get apiservice v1beta1.external.metrics.k8s.io -o yaml```

> If the status is False, then there seems to be an issue and proxy network might be the primary reason for it.

Reviewer: ```suggestion
If the status is `False`, then there seems to be an issue and proxy network might be the primary reason for it.
```

###### Follow these steps:

> Find the cluster IP for the keda-metrics-apiserver and keda-operator-metrics:

Reviewer: ```suggestion
Find the cluster IP for the `keda-metrics-apiserver` and `keda-operator-metrics`:
```

```kubectl get services --all-namespaces```

> In the ```/etc/kubernetes/manifests/kube-apiserver.yaml``` - add the cluster IPs found in the previous step in no_proxy variable.

Reviewer: ```suggestion
In the `/etc/kubernetes/manifests/kube-apiserver.yaml` - add the cluster IPs found in the previous step in no_proxy variable.
```

Reload systemd manager configuration:

```sudo systemctl daemon-reload```

Restart kubelet:

```sudo systemctl restart kubelet```

Check the API service status and the pods now. Should work!

> 

Reviewer: Nitpick - 2 new lines, let's remove one
```suggestion
```
